src/bp-moderation/classes/suspend/class-bp-core-suspend.php

```diff
@@ -420,9 +448,14 @@ public static function check_suspended_content( $item_id, $item_type ) {
 	 */
 	public static function check_user_suspend( $user_id ) {
 		global $wpdb;
-		$bp = buddypress();
+		$bp        = buddypress();
+		$cache_key = 'bb_check_user_suspend_' . $user_id;
```

Comment:
I think you have the setup key as wrong it will be like this:

```suggestion
		$cache_key = 'bb_check_user_suspend_user_' . $user_id;
```